test/unit/services/domain/test-capacity-date-range.js

```diff
@@ -44,180 +38,4 @@ describe('services/domain/capacity-date-range', function () {
     expect(capacityDateRange.toFields[1]).to.equal(VALID_TO_MONTH)
     expect(capacityDateRange.toFields[2]).to.equal(VALID_TO_YEAR)
   })
-
-  it('should throw ValidationError if a future date was given for capacity from date', function () {
```

Comment:
all of the actual validate functions are tested closer https://github.com/ministryofjustice/wmt-web/blob/error-list-in-controller/test/unit/services/validators/test-fieldset-validator.js, I didn't think it was worth duplicate tests testing the same underlying function